_assets/stylesheets/pages/_app.scss

```diff
@@ -7,14 +26,15 @@
     }
 
     .hero-transform {
```

Comment:
I know you didn't create this class but it should always be called a jumbotron to keep consistency